### DISCUSSION TITLE
Help with error in the db-RDA analysis - creating empty plot

### DISCUSSION THREAD

**Discussion by kaleklett** [ChiLiubio/microeco#125] [Q&A]
Hi!
I've been very much enjoying using the `microeco` package for my sequencing data, but I am currently stuck a bit in the `trans_env` class section. 

I am able to run the `cal_diff` and `cal_autocor` functions, but when I get to `cal_ordination` something goes wrong and eventually, the output that I get when I plot things is an empty plot:
[2022-07-19 (2)](https://user-images.githubusercontent.com/109585144/179710007-50b98739-c5ea-4611-b45c-b57f9ecbf8b8.png)

I'm not sure if this info helps detect a problem, but the output of my 

`object$res_ordination` is:
```Call: 
dbrda(formula = use_data ~ dryweight_plant + total_shoot_length + 
    total_berries + pH + soil_dry_matter + Nmin_soil + plant_N + 
    plant_P + soil_Ca, data = env_data)
Regularization parameters: 
NULL
Prior probabilities of groups: 
NULL
!Misclassification rate: 
       apparent:  %
```

and the output for x and y in my
`t_env$res_ordination_trans` is:

```
dryweight_plant | NaN | NaN |   |  
total_shoot_length | NaN | NaN |   |  
total_berries | NaN | NaN |   |  
pH | NaN | NaN |   |  
soil_dry_matter | NaN | NaN |   |  
Nmin_soil | NaN | NaN |   |  
plant_N | NaN | NaN |   |  
plant_P | NaN | NaN |   |  
soil_Ca | NaN | NaN
```

Is anyone else having similar problems? For some reason I was able to produce a plot previously before I updated microeco to the latest version, but it could also be that I have changed things somehow that is messing things up (I am still a beginner at using this package in R). I ran into the same problem processing both my bacterial and fungal dataset. 

For reference, I'm using:
R: 4.1.3
Microeco: 0.11.0

I would be thankful for any help or pointers in the right direction with how to figure out why it's not working. 
Cheers, Kristin

**Comment by ChiLiubio** [comment]
Hi @kaleklett 
Please see this Issue topic (https://github.com/ChiLiubio/microeco/issues/126) to save your microtable data and paste the compressed format here. Please also paste your scripts. I will try to run and check it to find what is the problem.

Chi

**Reply by kaleklett** [reply]
Hi!
I have sent you an email with more details. Thank you for getting back to me so quickly.
Best, Kristin

**Comment by ChiLiubio** [comment]
I have tried two times and get the result successfully.  I send you the result via email.

Best,
Chi
